addons/binding/org.openhab.binding.yamahareceiver/README.md

```diff
@@ -23,6 +23,26 @@ Bridge yamahareceiver:yamahaAV:ReceiverID "Yamaha Receiver Bridge Name" [host="a
 If your receiver is using menu-based net radio navigation, you can use this binding to
 select radio stations from a configured menu.
 
+Configuration parameters for Bridge type `yamahareceiver`:
+
+Parameter | Required | Default | Description
+--------|------|------|------
+`host` | yes | / | The IP address of the AVR to control
```

Comment:
I think it would be better to say `N/A` (not applicable) rather than just /. What do you think?

Reply:
Sounds good. Done.